container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -684,3 +684,30 @@ log to the console to aid debugging. Documentation for this object can be found
 
 You can also take a look at the ![](%%nblink _tutorials/Executing JavaScript Code Directly in SQL Queries Using the jseval Function Tutorial) for examples of how to use the `jseval` function.
 
+## <a name="try"></a>Handling errors line by line
+
+When a function is given invalid input, the whole query fails and no result is
```

Comment:
"When a function is given invalid input" -> When an error occurs when processing a query, the whole query fails and no result is given even if a single line caused the error.